momento-sdk/src/main/java/momento/sdk/CacheClient.java

```diff
@@ -1199,6 +1201,134 @@ public CompletableFuture<CacheDictionarySetFieldResponse> dictionarySetField(
     return scsDataClient.dictionarySetField(cacheName, dictionaryName, field, value, null);
   }
 
+  /**
+   * Sets all the fields in the given dictionary.
+   *
+   * @param cacheName - The cache containing the list.
+   * @param dictionaryName - The dictionary to set the field in.
+   * @param items - The fields to set.
+   * @param ttl Time to Live for the item in Cache. This ttl takes precedence over the TTL used when
+   *     building a cache client {@link CacheClient#builder(CredentialProvider, Configuration,
+   *     Duration)}
+   * @return Future containing the result of the dictionary fetch back operation.
+   */
+  public CompletableFuture<CacheDictionarySetFieldsResponse> dictionarySetFieldsStringString(
```

Comment:
I imagine this will be the one that we want to leave as `dictionarySetFields()`. @cprice404 do you agree?

Reply:
Yeah, taking care of that in a separate PR - https://github.com/momentohq/client-sdk-java/pull/219